modules/planning/tasks/deciders/path_reference_decider/path_reference_decider.cc

```diff
@@ -55,6 +55,11 @@ Status PathReferenceDecider::Execute(Frame *frame,
 
 Status PathReferenceDecider::Process(Frame *frame,
                                      ReferenceLineInfo *reference_line_info) {
+  // skip using path reference during lane changing
+  if (reference_line_info->size() > 1) {
```

Comment:
next time, if you haven't pushed this commit yet and find it's not valid, please revert it, instead of pushing an invalid commit, together with your upcoming commits.

Reply:
Fixed. Thank you.